proposed/phpdoc-tags.md

```diff
@@ -917,14 +897,10 @@ Each Constant or Property definition or Variable where the type is ambiguous
 or unknown SHOULD be preceded by a DocBlock containing the @var tag. Any
 other variable MAY be preceded with a DocBlock containing the @var tag.
 
-The @var tag MUST contain the name of the element it documents. An exception
+The @var tag SHOULD contain the name of the element it documents. An exception
```

Comment:
This confuses me. I think the `MUST` was better in this case. 
`@var` has to contain the element's name, otherwise the interpreter will never know which element is described. There is just one exception. In case it is used above a single property. 

Maybe add an example of multi property definition?